bookwyrm/templates/rss/edition.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+{% blocktrans trimmed with book_title=obj.title book_author=obj.author_text %}
+  ‘{{ book_title }}’ by {{ book_author }}
```

Comment:
Need to cover cases where there is no author listed. Something like:

`{{ book_title }}’{% if book_author %}by {{ book_author }}{% endif %}`

Reply:
handled in  98724dfa4774e2103e3a80b6b835959bef422ddaea68dce98b